style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightgrey;
+  font-family: sans-serif;
+}
+
+#section1 {
+  background-image: url(img/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
+  background-color: white;
+  background-size: 100%;
+  border: 2px solid white;
+  border-radius: 0 0 0 50px;
+}
+
+.first-list {
+  display: none;
+}
+
+.nav1 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: white;
+  padding-bottom: 1rem;
+  z-index: 1;
+}
+
+.text1 {
+  width: 100%;
+  padding-top: 4rem;
+}
+
+.logo {
+  margin: 1rem 0 0 3rem;
+  text-decoration: none;
+  font-size: 25px;
+  color: #6070ff;
+}
+
+.union {
+  margin: 1rem 3rem 0 0;
+}
+
+h1 {
+  font-size: 40px;
+  margin: 5rem 0 0 4rem;
+}
+
+.header-text {
+  color: #344563;
+  font-size: 25px;
+  margin: 1rem 0 0 4rem;
+  width: 70%;
+  line-height: 2.5rem;
+}
```

Comment:
- [ ] OPTIONAL: Header text looks huge compared to the original design, and the line height is too big, please consider decreasing the font size and line-height to match the design with figma: 

Your design  | Figma design
------------- | -------------
<img width="431" alt="Screenshot 2022-03-17 at 12 11 21" src="https://user-images.githubusercontent.com/77038610/158765188-d336165c-bcd8-47f7-84d4-0a489b6cc42d.png"> | <img width="436" alt="Screenshot 2022-03-17 at 12 11 54" src="https://user-images.githubusercontent.com/77038610/158765272-599fd822-3c9e-47f3-8a6f-9b93d9f8ae45.png">
- [ ] OPTIONAL: Please use the font-family provided in the design which is `Poppins`